shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/Alert.kt

```diff
@@ -10,13 +10,20 @@ data class Alert(
     @SerialName("active_period") val activePeriod: List<ActivePeriod>,
     val cause: Cause = Cause.UnknownCause,
     val description: String?,
+    @SerialName("duration_certainty")
+    val durationCertainty: DurationCertainty = DurationCertainty.Unknown,
     val effect: Effect = Effect.UnknownEffect,
     @SerialName("effect_name") val effectName: String?,
     val header: String?,
     @SerialName("informed_entity") val informedEntity: List<InformedEntity>,
     val lifecycle: Lifecycle,
     @SerialName("updated_at") val updatedAt: Instant
 ) : BackendObject {
+    init {
+        // This is done on init to avoid having to pass it in for any call to format an ActivePeriod
+        activePeriod.forEach { period -> period.durationCertainty = durationCertainty }
+    }
```

Comment:
thought (non-blocking): I looked a bit at alternative approaches for this since it feels a little funky for `durationCertainty on `ActivePeriod` to be nullable, but didn't find any clear cut options besides writing a custom serializer which seems like it could be more complicated than it is worth. 

It could be easier to pass durationCertainty as part of the activePeriods on the backend, but that may also be overkill. 

I think a comment in ActivePeriod about durationCertainty being null could help.

Reply:
Hah yeah, glad we were thinking along the same lines!